### PR TITLE
update text, english only (for now)

### PR DESCRIPTION
**Pull request by quaackk**
mmm punctuation, is this too many :/

### CODE REVIEW

File: data/text.json [hunk 1]
```diff
@@ -116,9 +116,9 @@
         "enchant":"`{0}enchant <text>` *turns text into the Minecraft enchantment table language*",
         "unenchant":"`{0}unenchant <text>` *turns the enchantment table language back into text*",
         "villagerspeak":"`{0}villagerspeak <text>` *turns text into villager sounds*",
-        "clap":"`{0}clap <text>` *:clap: puts :clap: the :clap: emoji :clap: between :clap: text*",
+        "clap":"`{0}clap <text>` *:clap: puts :clap: the :clap: clap :clap: emoji :clap: between :clap: text :clap:*",
```
Comment (Iapetus-11) [inline data/text.json]: This was intentional but ok

File: data/text.json [hunk 2]
```diff
@@ -229,13 +229,13 @@
...
         "zombie":{
           "attacks":[
-            "The **zombie** ran, lashing at your arms",
-            "The **zombie** gives you a painful hug",
-            "The **zombie** groans and gives you a painful hug",
-            "The **zombie** punches you in your face",
-            "The **zombie** claws you with its overgrown, decaying nails",
-            "The **zombie** punches you in your face with its decaying hand",
-            "The **zombie** scratches you with its overgrown, decaying nails"
+            "The **zombie** ran, lashing at your arms.",
```
Comment (Iapetus-11) [inline data/text.json]: Perhaps use exclamation points here?

File: data/text.json [hunk 3]
```diff
@@ -245,10 +245,10 @@
         },
         "skeleton":{
           "attacks":[
-            "The **skeleboi** yeets an arrow at your face",
+            "The **skeleboi** yeets an arrow at your face.",
```
Comment (Iapetus-11) [inline data/text.json]: ! here too

File: data/text.json [hunk 4]
```diff
@@ -260,28 +260,28 @@
         },
         "spider":{
           "attacks":[
-            "The **spider** jumps on top of you and sinks its fangs into your arm",
-            "The **spider** knocks you down and bites your leg",
-            "The **spider** jumps at you in an attempt to bite your face off",
-            "The **spider** dunks on you",
-            "The **spider** lunges at you and bites your leg"
+            "The **spider** jumps on top of you and sinks its fangs into your arm.",
```
Comment (Iapetus-11) [inline data/text.json]: and here

File: data/text.json [hunk 5]
```diff
@@ -260,28 +260,28 @@
...
+            "The **spider** knocks you down and bites your leg.",
+            "The **spider** jumps at you in an attempt to bite your face off.",
+            "The **spider** dunks on you.",
+            "The **spider** lunges at you and bites your leg."
           ],
           "finishers":[
             "The **spider** jumps on you, spins around, and sinks its fangs into your skull!",
             "The **spider** dunks on you one last time, and bites your face off!",
-            "The **spider** rips your face off, your last sight being 8 murderous red eyes staring back"
+            "The **spider** rips your face off, your last sight being 8 murderous red eyes staring back."
```
Comment (Iapetus-11) [inline data/text.json:272]: and here

File: data/text.json [hunk 6]
```diff
@@ -260,28 +260,28 @@
...
           ]
         },
         "cave_spider":{
           "attacks":[
-            "The **cave spider**'s fangs digs deep onto your skin",
-            "The **cave spider** gives you a toxic kiss",
-            "The **cave spider** bites you and inflicts poison damage"
+            "The **cave spider**'s fangs digs deep onto your skin.",
```
Comment (Iapetus-11) [inline data/text.json]: and here

File: data/text.json [hunk 7]
```diff
@@ -260,28 +260,28 @@
...
+            "The **cave spider** gives you a toxic kiss.",
+            "The **cave spider** bites you and inflicts poison damage."
           ],
           "finishers":[
-            "You died a slow, painful death from poison damage",
-            "The **cave spider** injects one last lethal dose of venom",
-            "The **cave spider** buries its fangs deep into you and fills you with venom"
+            "You died a slow, painful death from poison damage.",
```
Comment (Iapetus-11) [inline data/text.json]: and here

File: data/text.json [hunk 8]
```diff
@@ -301,24 +301,24 @@
         },
         "baby_slime":{
           "attacks":[
-            "The **baby slime** jumps on you",
-            "The **baby slime** looks cute",
-            "The **baby slime** jumps on your head",
-            "You encourage the **baby slime**"
+            "The **baby slime** jumps on you.",
```
Comment (Iapetus-11) [inline data/text.json]: and here

**Review by Iapetus-11 — CHANGES_REQUESTED** [review]

**Comment by quaackk** [comment]
that should be all of them

File: data/text.json [hunk 9]
```diff
@@ -249,7 +249,7 @@
             "The **skeleton** turns around and 360 no scopes you!",
             "An arrow rains from the **skeleton**, only one.",
             "One headshot from the **skeleton** for you!",
-            "BOOM! HEADSHOT! - **skeleton** 2021"
+            "BOOM! HEADSHOT! - **skeleton 2021**"
```
Comment (Iapetus-11) [inline data/text.json]: This was correct the way it was.

**Review by Iapetus-11 — CHANGES_REQUESTED** [review]

**Comment by quaackk** [comment]
i swear, ok is this everything now

**Review by Iapetus-11 — APPROVED** [review]

**Pull request merged**